Doc/reference/compound_stmts.rst

```diff
@@ -383,6 +383,7 @@ have ambiguous semantics.
 
 It is not possible to mix :keyword:`except` and :keyword:`!except*`
 in the same :keyword:`try`.
+The statements
```

Comment:
```suggestion
The keywords
```
Is a bit more specific.

But either way, I agree adding the words makes the text easier to read.

Reply:
While yes, they are keywords, the core issue demanding they cannot be used is because those three are _flow-control **statements**_ that would mess up processing of multiple `except*` clauses.

Reply:
In that sense, adding the two extra words ("flow-control"; hyphenated?) might provide even more clarity.